index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to create your unique account on [Formspree service (Click here for more)](https://formspree.io/html). the `{form_id}` is just the placeholder after you get your unique `id` paste it there instead of the placeholder which will allow us to submit the form successfully.